FileIO/CMakeLists.txt

```diff
@@ -61,6 +61,12 @@ target_link_libraries(FileIO INTERFACE
 )
 if(QT4_FOUND)
 	target_link_libraries(FileIO PUBLIC Qt4::QtXml Qt4::QtXmlPatterns)
+	if(WIN32 AND CMAKE_CROSSCOMPILING)
+		find_package(OpenSSL)
+		if(OPENSSL_FOUND)
```

Comment:
We don't need SSL but the prebuilt Qt package from http://pkg.mxe.cc is built against it which means that `QtNetwork` depends on it (and `QtXmlPatterns` (containing the XML validator) depends on `QtNetwork`).